src/core/stores/VNodes/CharNode.ts

```diff
@@ -0,0 +1,10 @@
+import { VNode, VNodeType, FormatType } from '../VNode';
+
+export class CharNode extends VNode {
+    properties = {
+        atomic: true,
+    };
+    constructor(char: string, format: FormatType) {
+        super(VNodeType.CHAR, '#text', char, format);
```

Comment:
I suggest renaming the `value` property of `VNode` to `name` such that we could merge `originalTag` and `value` since `value` is not used for elements and `originalTag` is this useless `#text` for text nodes. The render function of element nodes would know that `name` is the `nodeName` while the render function of char nodes would know that `name`, in the context of a char node, is the character itself. Maybe this `name` property could then go into the `properties` dict ?